### PR TITLE
update for boba.

### PR DESCRIPTION
**Pull request by Tulun**
View for BOBA added to dropdown.

Note: transaction untested, but this is a separate PR. If we want to merge this to master, we just comment out BOBA from the dropdown.

### CODE REVIEW

**Review by daywiss — COMMENTED** [review]
looks good, but found an issue when selecting boba. This says we have boba: 
image
but when trying to send it, the balance is 0.
image

**Comment by daywiss** [comment]
getting a call reverted when i try to send eth from boba: 
image

**Comment by daywiss** [comment]
looks like weth address for boba might be wrong 
image

File: src/state/send.ts [hunk 1]
```diff
@@ -27,7 +27,8 @@ const initialState: State = {
   amount: ethers.constants.Zero,
   toChain: DEFAULT_TO_CHAIN_ID,
   fromChain: DEFAULT_FROM_CHAIN_ID,
-  currentlySelectedToChain: CHAINS_SELECTION[2],
+  // Default to ethereum, which should be end of this array.
+  currentlySelectedToChain: CHAINS_SELECTION[CHAINS_SELECTION.length - 1],
```
Comment (Gamaranto) [inline src/state/send.ts:31]: just a nit, is there a way we could type this so TS will warn us if Ethereum is not the last element in the array?

**Review by Gamaranto — APPROVED** [review]

**Comment by Tulun** [comment]
> looks like weth address for boba might be wrong image

Do they have a WETH address? I couldn't find it in their docs.

File: src/state/send.ts (hunk 1 above)
Comment (Tulun) [reply src/state/send.ts:31]: I imagine if we wanna make this into a typed Tuple :)

File: src/utils/constants.ts [hunk 2]
```diff
@@ -280,20 +280,20 @@ export const TOKENS_LIST: Record<ChainId, TokenList> = {
...
       bridgePool: getAddress("0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48"),
     },
+    // {
+    //   address: getAddress("0xa18bF3994C0Cc6E3b63ac420308E5383f53120D7"),
+    //   name: "Boba",
+    //   symbol: "BOBA",
+    //   decimals: 18,
+    //   logoURI: bobaLogo,
+    //   bridgePool: getAddress("0x42bbfa2e77757c645eeaad1655e0911a7553efbc"),
+    // },
```
Comment (daywiss) [inline src/utils/constants.ts:290]: does this need to go in?

File: src/components/CoinSelection/CoinSelection.tsx [hunk 3]
```diff
@@ -174,6 +174,7 @@ const CoinSelection = () => {
     (fees?.isAmountTooLow && amount.gt(0)) ||
     (fees?.isLiquidityInsufficient && amount.gt(0));
 
+  console.log("tokensList", tokenList, balances);
```
Comment (daywiss) [inline src/components/CoinSelection/CoinSelection.tsx:177]: stray log

**Review by daywiss — APPROVED** [review]

**Comment by daywiss** [comment]
approved, but saw this in console 
image

**Pull request merged**